packages/react-components/src/organisms/MembersSection.tsx

```diff
@@ -45,12 +45,15 @@ const roleStyles = css({
 });
 
 type MembersSectionProps = {
-  readonly members: TeamMember[];
+  readonly title: string;
+  readonly members: (Omit<TeamMember, 'role'> & {
+    role: string;
```

Comment:
What is happening here, this doesn't look right - is the component not able to deal with the specific subtype of string for some reason?

Reply:
The role on user.teams limits to a closed list. The role on the discover page isn't on that list.
This property change removes the constraints on the presentation part (on members component) to accommodate the hardcoded role on discover.